torch/autograd/grad_mode.py

```diff
@@ -386,12 +386,13 @@ class _unsafe_preserve_version_counter(_DecoratorContextManager):
 
     """
 
-    def __init__(self, tensor: torch.Tensor) -> None:
-        self.tensor = tensor
-        self.prev_version = tensor._version
+    def __init__(self, tensors: Union[torch.Tensor, List[torch.Tensor]]) -> None:
```

Comment:
```suggestion
    def __init__(self, tensors: Union[torch.Tensor, Tuple[torch.Tensor, ...]]) -> None:
```

Reply:
You use both in your tests :p I would settle on tuple unless we often need the flexibility of a list.

Reply:
fair, will do